pkg/label/filter.go

```diff
@@ -0,0 +1,96 @@
+package label
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/Peripli/service-manager/pkg/types"
+
+	"github.com/Peripli/service-manager/pkg/query"
+	"github.com/Peripli/service-manager/pkg/util"
+	"github.com/Peripli/service-manager/pkg/web"
+	"github.com/tidwall/gjson"
+)
+
+// DenyLabelOperations describe denied label key operations
+type DenyLabelOperations map[string][]query.LabelOperation
+
+type FilterOperations struct {
+	deniedOperations DenyLabelOperations
+}
+
+func NewFilterOperations(deniedOperations DenyLabelOperations) *FilterOperations {
+	return &FilterOperations{
+		deniedOperations: deniedOperations,
+	}
+}
+
+func (flo *FilterOperations) Name() string {
+	return "FilterOperations"
+}
+
+func (flo *FilterOperations) Run(req *web.Request, next web.Handler) (*web.Response, error) {
+	if req.Method == http.MethodPost {
+		bodyMap := gjson.ParseBytes(req.Body).Map()
```

Comment:
No need to include the variable type into it's name - you can just use `body` and `labels` instead of `bodyMap` and `labesBytes` - the type of the variable is clear from the declaration.